backend/connector_processor/connector_processor.py

```diff
@@ -128,6 +129,9 @@ def test_connectors(connector_id: str, cred_string: dict[str, Any]) -> bool:
         except ConnectorError as e:
             logger.error(f"Error while testing {connector_id}: {e}")
             raise TestConnectorInputException(core_err=e)
+        except KeyError as e:
+            logger.error(f"Path not found while testing {connector_id}: {e}")
+            raise KeyNotFoundException(message=str(e))
```

Comment:
I don't think we should be catching and raising for such pythonic exceptions - they can be treated as 500 errors with some generic message itself. We should technically prevent some KeyErrors during dict accesses

Reply:
removed @chandrasekharan-zipstack . We already have a generic message here[https://github.com/Zipstack/unstract/blob/main/backend/connector_processor/exceptions.py#L42](url)